<launch>

 <!--pickup and dropoff zone coordinates--> 
  <param name = "goal_pickup1_x" value = "-7.0"/>
  <param name = "goal_pickup1_y" value = "-4.0"/>
  <param name = "goal_pickup2_x" value = "14.0"/>
  <param name = "goal_pickup2_y" value = "0.0"/>
  <param name = "goal_dropoff1_x" value = "13.2"/>
  <param name = "goal_dropoff1_y" value = "8.3"/>
  <param name = "goal_origin_x" value = "0.0"/>
  <param name = "goal_origin_y" value = "0.0"/>
 
 <!-- The add_markers node for home service -->
  <node name="add_markers_hs" type="add_markers_hs" pkg="add_markers" output="screen">
  </node>

</launch>
